core/src/main/resources/jenkins/model/Jenkins/help-rawBuildsDir.html

```diff
@@ -1,15 +1,21 @@
 <div>
-    Specify where Jenkins would store records of the past builds.
-    This value can include the following variables.
-
+    Specify where Jenkins should store build records. This includes the console
+    output and other metadata generated by a build.
+    <p>
+    This value can include the following variables:
     <ul>
-        <li><tt>${JENKINS_HOME}</tt> &mdash; Jenkins home directory.
-        <li><tt>${ITEM_ROOTDIR}</tt> &mdash; Root directory of a job for which the default workspace is allocated.
-        <li><tt>${ITEM_FULL_NAME}</tt> &mdash; '/'-separated job name, like "foo/bar".
+        <li><tt>${JENKINS_HOME}</tt> &mdash; The Jenkins home directory
+        <li><tt>${ITEM_ROOTDIR}</tt> &mdash; The directory where Jenkins stores config data for a given job
+        <li><tt>${ITEM_FULL_NAME}</tt> &mdash; The full name of a given job,
+            which may be slash-separated, e.g. <tt>foo/bar</tt> for the job
+            <tt>bar</tt> in folder <tt>foo</tt>
     </ul>
-
+    The value must include <tt>${ITEM_ROOTDIR}</tt> or
+    <tt>${ITEM_FULL_NAME}</tt>, so that each job can store its build records
+    separately.
+    <p>
+    Changing this value allows you to store build records on a larger, but
```

Comment:
It's important to tell people that Jenkins doesn't move anything on its own when this option is changed. (So IMO it's a weird option to have on the UI \*sigh\*).